business_register/pep_scoring/rules.py

```diff
@@ -0,0 +1,108 @@
+from abc import ABC, abstractmethod
+import json
+from business_register.models.declaration_models import (Declaration,
+                                                         Property,
+                                                         Vehicle,
+                                                         VehicleRight,
+                                                         Income,
+                                                         Money,
+                                                         PropertyRight,
+                                                         )
+from business_register.models.pep_models import (RelatedPersonsLink, Pep)
+from location_register.models.ratu_models import RatuCity
+
+
+class BaseScoringRule(ABC):
+    def __init__(self, pep):
+        self.pep = pep
+
+    @abstractmethod
+    def calculate_weight(self):
+        pass
+
+
+class IsRealEstateWithoutValue(BaseScoringRule):
+    """
+    Rule 3.1 - PEP03_home
+    weight - 0.4
+    There is no information on the value of the real estate owned by PEP or
+    family members since 2015
+    """
+
+    def calculate_weight(self):
+        family_ids = self.pep.related_persons.filter(
+            to_person_links__category=RelatedPersonsLink.FAMILY,
+        ).values_list('id', flat=True)[::1]
+        family_ids.append(self.pep.id)
+        have_weight = PropertyRight.objects.filter(
+            pep_id__in=family_ids,
+            property__valuation__isnull=True,
+            type=Property.SUMMER_HOUSE,
+            acquisition_date__year__gte=2015,
+        ).values_list('property_id', 'property__declaration_id')[::1]
+
+        if have_weight:
+            weight = 0.4
+            value = {
+                "property_id": have_weight[0][0],
+                "declaration_id": have_weight[0][1],
+            }
+            return json.dumps(value), weight
+        return 0
+
+
+class IsLandWithoutValue(BaseScoringRule):
+    """
+    Rule 3.2 - PEP03_land
+    weight - 0.1
+    There is no information on the value of the land owned by PEP or
+    family members since 2015
+    """
+
+    def calculate_weight(self):
+        family_ids = self.pep.related_persons.filter(
+            to_person_links__category=RelatedPersonsLink.FAMILY,
+        ).values_list('id', flat=True)[::1]
+        family_ids.append(self.pep.id)
+        have_weight = PropertyRight.objects.filter(
+            pep_id__in=family_ids,
+            property__valuation__isnull=True,
+            type=Property.LAND,
+            acquisition_date__year__gte=2015,
+        ).values_list('property_id', 'property__declaration_id')[::1]
+        if have_weight:
+            weight = 0.1
+            value = {
+                "property_id": have_weight[0][0],
+                "declaration_id": have_weight[0][1],
+            }
+            return json.dumps(value), weight
+        return 0
```

Comment:
1. no need to call `json.dumps`
2. function should return same result if don't `have_weight` 
like this ->
```suggestion
            return weight, data
        return 0, {}
```